docker-compose.prod.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+
+services:
+  codex-accounting-prod:
+    build:
```

Comment:
Please, copy from there `https://github.com/codex-team/hawk.api.nodejs/blob/master/docker-compose.prod.yml`.
You need network: host